src/formattedtext.jl

```diff
@@ -232,6 +253,22 @@ function layout_formatted_text(
                   "is of type '$(typeof(textelement))'")
         end
 
+        # replace emojis here which are given by the syntax :emoij_shorthand:
+        textelement_emojis = Tuple{String,Int64}[]
+        for m in eachmatch(RGX_EMOJI, textelement_string)
+            # query shorthands and replace them with a placeholder character
+            # later we plot the actual emoji ontop of the placeholder
+            e = first(m.captures)
+            !(e in keys(EMOJIS_MAP)) && error("Unknown emoij shorthand '$e'")
```

Comment:
Should we warn here instead, and simply not render it (or render a box)?

Reply:
Oh, yes, we should.

Reply:
I am  now inserting � if the emoji is not recognized and then skip rendering of that part.